core/jazz_ui/e2e/page-objects/jazzservices.po.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ *Copyright 2016-2017 T Mobile, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); You may not use
+ * this file except in compliance with the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser, by, element, $, $$ } from 'protractor';
+const fs = require('fs');
+const applicationDomain = '';
+
+export class Jazz {
+
+  navigateToJazzGet() {
+    return browser.driver.get(applicationDomain);
+  }
+
+  getCreateService() {
+    return element(by.xpath('//div[@class="page-title-wrap hide-for-mobile"]/btn-primary-with-icon[@text="CREATE SERVICE"]'));
```

Comment:
Can we select elements by css properties instead of xpath.
Ref: [https://blog.jscrambler.com/getting-started-with-angular-2-end-to-end-testing/](https://blog.jscrambler.com/getting-started-with-angular-2-end-to-end-testing/)

Reply:
Since its works with xpath, Let it be.
IF I get some time after the user stories we can estimate the time and will do.

Reply:
As per the discussion,we are going to move that as a separate user story

Reply:
Yes we should do this refactor at some point.